comid/class.go

```diff
@@ -0,0 +1,153 @@
+package comid
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+// TODO(tho) OID
+
+// Class represents the class of the (target / attesting) environment.  The only
+// required field is the class unique identifier (see ClassID).  Optionally,
```

Comment:
is line 15 strictly required, I mean the class identifier.

I thought this is also optional, as the device class could also be mapped using V,M.L and I.